source/NVDAObjects/window/edit.py

```diff
@@ -840,7 +840,7 @@ def _getEmbeddedObjectLabel(self, embedRangeObj):
 			dataObj = None
 		if dataObj:
 			text = comtypes.BSTR()
-			res = NVDAHelper.localLib.getOleClipboardText(dataObj, ctypes.byref(text))  # noqa: F841
+			NVDAHelper.localLib.getOleClipboardText(dataObj, ctypes.byref(text))
```

Comment:
should we be checking the result here and logging a failure?

Reply:
I do not know how frequently this call can fail.
If I log an error, is there a risk that I log many errors here?

Also, looking at function `cancellableSendMessageTimeout` (in file `nvdaHelperLocal.cpp`), it's not clear to me if I should only look at return code 0 and look at last error, or if some other non-zero return codes also need to be logged in the Python side of the code.